Examples/HackerNewsClient/HackerNewsClient/UI/HackerNews.swift

```diff
@@ -0,0 +1,192 @@
+                        .highlighted: UIColor(white: 1.0, alpha: 0.5)
+                    ]
+                    $0.backgroundColor = .clear
+                    $0.isEnabled = !props.loading
+                    $0.alpha = props.loading ? 0 : 1
+                    $0.touchHandlers = [
+                        .touchUpInside: reload
+                    ]
+                })),
+                Table(props: Table.Props.build({
+                    $0.setKey(Keys.table)
+                    $0.delegate = TableViewDelegate(posts: props.posts)
+                })),
+                Label(props: Label.Props.build({
+                    $0.setKey(Keys.emtpyText)
+                    $0.text = NSAttributedString(string: "Loading some posts...", attributes: [
+                        NSForegroundColorAttributeName: UIColor.gray,
+                        NSFontAttributeName: UIFont.systemFont(ofSize: 17)
+                        ])
+                    $0.textAlignment = NSTextAlignment.center
+                })),
+            ]
+        
+        if props.openPostURL != nil {
+            let browser = Browser(props: Browser.Props.build({
+                $0.setKey(Keys.browser)
+                $0.url = props.openPostURL
+                $0.alpha = (props.openPostURL == nil) ? 0 : 1
+            }))
+
+            nodes.append(browser)
+        }
+        
+        return nodes
+    }
+    
+    static func layout(views: ViewsContainer<Keys>,
+                       props: PropsType,
+                       state: StateType) {
+        let root = views.nativeView
+        let titleLabel =  views[.titleLabel]!
+        let refreshButton = views[.reloadButton]!
+        let spinner = views[.spinner]!
+        let table = views[.table]!
+        let emptyText = views[.emtpyText]!
+        let browser = views[.browser]
+        
+        titleLabel.height = .scalable(80)
+        titleLabel.asHeader(root, insets: .scalable(30, 0, 0, 0))
+        
+        refreshButton.fillVertically(titleLabel)
+        refreshButton.width = .scalable(100)
+        refreshButton.setRight(titleLabel.right, offset: .scalable(-10))
+        
+        spinner.center(refreshButton)
+        
+        table.fillHorizontally(root)
+        table.top = titleLabel.bottom
+        table.bottom = root.bottom
+        
+        if props.posts.isEmpty {
+            emptyText.fill(table)
+        }
+        
+        browser?.fill(root)
+    }
+    
+    static func updateChildrenAnimations(container: inout ChildrenAnimations<ChildrenKeys>,
+                                         currentProps: PropsType,
+                                         nextProps: PropsType,
+                                         currentState: StateType,
+                                         nextState: StateType) {
+        container[.browser] = Animation(
+            type: .linear(duration: 0.3),
+            entryTransformers: [AnimationProps.moveRight(distance: 320), AnimationProps.fade()],
```

Comment:
the distance is not scaled like in Plastic. So I suggest you use `props.frame` instead of a plain number